ct_optcon/examples/plotResultsPointMass.h

```diff
@@ -50,8 +50,8 @@ void plotResultsPointMass(const ct::core::StateVectorArray<STATE_DIM>& stateArra
         Eigen::VectorXd x, y;
         x = r*theta.array().sin().matrix();
         y = r*theta.array().cos().matrix();
-        x = x + x_obst;
-        y = y + y_obst;
+        x.array() = x.array() + x_obst;
+        y.array() = y.array() + y_obst;
```

Comment:
@hmcm this needs to be done using array operations, otherwise it will not compile using many eigen versions.